webapp/channels/src/components/global_header/left_controls/product_menu/product_menu_list/__snapshots__/product_menu_list.test.tsx.snap

```diff
@@ -501,6 +501,7 @@ exports[`components/global/product_switcher_menu should show integrations should
   <li
     className="MenuGroup menu-divider"
     onClick={[Function]}
+    role="presentation"
```

Comment:
Shouldn't this be separator ? The separator role indicates that an element is a divider, while the presentation role indicates that an element is being used only for presentation.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/separator_role